### PR TITLE
[backend] config update

### PR DESCRIPTION
**Pull request by ann-aot**
1. Added a method in the config service to retrieve the process env variable for test-db-seeding

### CODE REVIEW

File: sources/packages/backend/apps/test-db-seeding/src/main.ts [hunk 1]
```diff
@@ -3,19 +3,24 @@ import { NestFactory } from "@nestjs/core";
...
 import { SeedExecutor } from "./seed-executors/seed-executor";
+import { ConfigService } from "@sims/utilities/config";
 // Clean db command line identifier.
 const CLEAN_DB = "task=clean-db";
 // Filter classes to be seeded command line identifier.
 const FILTER_CLASSES = "filter=";
 // QA DB name. for safe DB clean.
 const QA_DB_NAME = "QASIMSDB";
+// Config instance.
+const configService = new ConfigService();
```
Comment (andrewsignori-aot) [inline sources/packages/backend/apps/test-db-seeding/src/main.ts:14]: The idea is always to use the injected one unless there is a problem.
```ts
const configService app.get(ConfigService);
```

File: sources/packages/backend/libs/utilities/src/config/config.service.ts [hunk 2]
```diff
@@ -183,6 +184,15 @@ export class ConfigService {
     });
   }
 
+  /**
+   * Database seeding for testing configuration.
+   */
+  get testDatabaseSeeding(): TestDatabaseSeedingConfig {
```
Comment (andrewsignori-aot) [inline sources/packages/backend/libs/utilities/src/config/config.service.ts:190]: The configuration here should be the generic one for the DB connection.
It it is not part of the PR to create all DB properties now we should at least create this one as part of an object that would map them all.

**Review by andrewsignori-aot — COMMENTED** [review]
Thanks for going back and doing these changes @ann-aot. Please take a look at the comments.

File: sources/packages/backend/apps/test-db-seeding/src/main.ts [hunk 3]
```diff
@@ -3,19 +3,24 @@ import { NestFactory } from "@nestjs/core";
...
 
 async function bootstrap() {
   const app = await NestFactory.create(TestDbSeedingModule);
   // Checking for CLEAN_DB parameter.
   if (process.argv.includes(CLEAN_DB)) {
     // Clean db.
-    if (process.env.POSTGRES_DB?.includes(QA_DB_NAME)) {
+    if (
+      configService.testDatabaseSeeding.qaDatabaseName?.includes(QA_DB_NAME)
```
[the remark this answers is not in the document]
Comment (ann-aot) [reply sources/packages/backend/apps/test-db-seeding/src/main.ts:22]: actually, it's part of my old PR. and the idea was not to strictly compare

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the changes, looks good 👍

File: sources/packages/backend/apps/test-db-seeding/src/main.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (andrepestana-aot) [reply sources/packages/backend/apps/test-db-seeding/src/main.ts]: ok. But did you choose not to compare strictly here?

**Review by andrepestana-aot — APPROVED** [review]
LGTM
